public-api/src/main/java/org/pmiops/workbench/cdr/model/DbConcept.java

```diff
@@ -38,13 +38,15 @@
     private String drugBrandNames;
     private int canSelect;
     private int hasCounts;
-    private MeasurementConceptInfo measurementConceptInfo = null;
+    private DbMeasurementConceptInfo dbMeasurementConceptInfo = null;
+    private String graphToShow = "Sex Assigned at Birth";
+
+    public DbConcept() {
 
-    public Concept() {
     }
 
     // Copy constructor for copying everything but synonyms
-    public Concept(Concept a) {
+    public DbConcept(DbConcept a) {
```

Comment:
nit: this constructor in never used. Can it be removed? Or at least remove the transient properties `graphToShow` that isn't being referenced.

Reply:
Removed the graph property.